styles/status.cafe/catppuccin.user.css

```diff
@@ -0,0 +1,189 @@
+        @accent-color: @catppuccin[@@lookup][@@accent];
+
+        color-scheme: if(@lookup = latte, light, dark);
+
+        ::selection {
+            background-color: fade(@accent-color, 30%);
+        }
+
+        input,
+            textarea {
+                &::placeholder {
+                color: @subtext0 !important;
+            }
+        }
+
+        background-color: @base;
+
+        body {
+            background-color: @base;
+            color: @text;
+        }
+        
+        textarea {
+            background-color: @mantle;
+            color: @text;
+            border-color: @surface1;
+            border-style: solid;
+            border-width: 1px;
+            border-radius: 2px;
+            padding: 3px;
+        }
+        
+        input {
+            background-color: @mantle;
+            color: @text;
+            border-color: @surface1;
+            border-style: solid;
+            border-width: 1px;
+            border-radius: 4px;
+            padding: 2px 5px;
+        }
+        
+        textarea:hover, input:hover {
+            border-color: darken(@surface1, 5%);
+        }
+        
+        input:hover {
+            background-color: darken(@mantle, 2%);
+        }
+        
+        input:active {
+            background-color: @crust;
+            border-color: @surface0;
+        }
+
+        .flash {
+            background-color: @green;
+        }
+
+        a:not(.flash a) {
+            color: @accent-color;
+        }
+
+        a:visited:not(.flash a) {
+            color: darken(@accent-color, 5%);
+        }
+        
+        table, th, td {
+            border-color: @surface1 !important;
+        }
+        
+        thead {
+            background-color: @surface1;
+            color: @text;
+        }
+        
+        .forum {
+            background-color: @yellow;
+            a {
+                color: darken(@blue, 10%) !important;
+            }
+        }
+        
+        tbody, .topic > tbody > tr:nth-child(2n) {
+            background-color: @mantle;
+        }
+        
+        hr {
+            background-color: @surface1;
+        }
+        
+        .signature {
+            border-top-color: @surface0;
+        }
+        
+        & when (@lookup = latte){
+            .forum {
+                background-color: lighten(@yellow, 15%);
+            }
+            table, th, td {
+                border-color: @text !important;
+            }
+            thead {
+                background-color: @text;
+                color: @base;
+            }
+            flash {
+                color: darken(@green, 15%);
+                a {
+                    color: @blue;
+                }
+            }
+            a:visited:not(.flash a) {
+                color: darken(@accent-color, 15%);
+            }
+            input:active {
+                background-color: @surface0;
+                border-color: @surface2;
+            }
+        }
```

Comment:
Why do these need to be different on latte?

Reply:
Mostly contrast issues or better visibility. The ones that target the forum (`.forum`, `table`, `th`, etc) are so that the theme looks similar to how the unthemed forum.status.cafe looks as I sadly couldn't make the dark-mode themes look good with that specific latte setup.

Reply:
It looks fine on latte without these latte overrides to me. What contrast issues were you looking at specifically?

Reply:
Mostly the visited links (so that they're a bit more visible across all the accent colors) and the input (so that the active state is also a bit more visible). I'm already thinking about removing the alteration to the `flash` but do you think I should also remove the forum related ones?

Reply:
@uncenter

Reply:
> Mostly the visited links (so that they're a bit more visible across all the accent colors) and the input (so that the active state is also a bit more visible). I'm already thinking about removing the alteration to the `flash` but do you think I should also remove the forum related ones?

Yeah that would be great, we should avoid modifying colors like this whenever possible.